format/mbr/testdata/mbr.fqtest

```diff
@@ -0,0 +1,9 @@
+# head -c 512 /dev/sda > mbr.bin
+$ fq . -d mbr mbr.bin
+     |00 01 02 03 04 05 06 07 08 09 0a 0b 0c 0d 0e 0f|0123456789abcdef|.{}: mbr.bin (mbr)
+0x000|fa bc 00 7c 31 c0 8e d0 8e c0 8e d8 52 be 00 7c|...|1.......R..||  code_area: raw bits
+*    |until 0x1bd.7 (446)                            |                |
+0x1b0|                                          00 00|              ..|  partition_table: raw bits
+0x1c0|00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00|................|
+*    |until 0x1fd.7 (64)                             |                |
+0x1f0|                                          55 aa|              U.|  boot_record_sig: raw bits
```

Comment:
This is always 0x55aa for a mbr? if so could seek and assert i guess

Reply:
Yes, I'll do that